frontend/src/Movie/MovieFileStatus.js

```diff
@@ -25,11 +26,10 @@ function MovieFileStatus(props) {
     return (
       <div className={styles.center}>
         <Label
-          title={queueStatusText}
+          title=" "
           kind={kinds.QUEUE}
```

Comment:
To avoid making changes to bypass validation on `Label` component, I think `InfoLabel` will work for these. Otherwise looks good

Reply:
Reworked with just css